docker-cli.yaml

```diff
@@ -67,3 +67,4 @@ update:
     identifier: docker/cli
     strip-prefix: v
     tag-filter: v
+    use-tag: true
```

Comment:
can we please ignore the rc versions in the same PR :-)

Reply:
added via `ignore-regex-pattern`, ptal and lmk if that was wrong/dumb